### PR TITLE
chromium: update to 127.0.6533.99

### LINKED ISSUE #7275: Google Chrome, Chromium: security update to 127.0.6533.72
### Affected package (and version)

`google-chrome` (126.0.6478.182), `chromium` (126.0.6478.182)

### CVE ID(s)

CVE-2024-6988, CVE-2024-6989, CVE-2024-6991, CVE-2024-6994, CVE-2024-6995, CVE-2024-6996, CVE-2024-6997, CVE-2024-6998, CVE-2024-6999, CVE-2024-7000, CVE-2024-7001, CVE-2024-7003, CVE-2024-7004, CVE-2024-7005

### Severity

High

### Other security advisory ID(s）

- [Tracking bug for internal fixes: Chrome 127, release 0](https://issues.chromium.org/issues/354788491)

### Description/References

- [Stable Channel Update for Desktop](https://chromereleases.googleblog.com/2024/07/stable-channel-update-for-desktop_23.html)

### Patch(es)/Solution(s)

Update to 127.0.6533.72.

### LINKED ISSUE #7424: chromium, google-chrome: security update to 127.0.6533.88
### Affected package (and version)

`chromium`, `google-chrome`: < 127.0.6533.88

### CVE ID(s)

CVE-2024-6990, CVE-2024-7255, CVE-2024-7256

### Severity

0-day

### Other security advisory ID(s）

N/A

### Description/References

See https://chromereleases.googleblog.com/2024/07/stable-channel-update-for-desktop_30.html.

### Patch(es)/Solution(s)

Update affected packages.

### LINKED ISSUE #7550: chromium, google-chrome: security update to 127.0.6533.99
### Affected package (and version)

`google-chrome`, `chromium` (<127.0.6533.99)

### CVE ID(s)

CVE-2024-7532, CVE-2024-7533, CVE-2024-7550, CVE-2024-7534, CVE-2024-7535, CVE-2024-7536

### Severity

Critical

### Other security advisory ID(s）

- [Stable Channel Update for Desktop](https://chromereleases.googleblog.com/2024/08/stable-channel-update-for-desktop.html)

### Description/References

- [Stable Channel Update for Desktop](https://chromereleases.googleblog.com/2024/08/stable-channel-update-for-desktop.html)

### Patch(es)/Solution(s)

Update to 127.0.6533.99.

### PR DESCRIPTION
**Pull request by KexyBiscuit**
Topic Description
-----------------

- chromium: bump REL for topic Revision Marking Guidelines
- libyuv: bump REL for topic Revision Marking Guidelines
- chromium: try to fix build fails
- chromium: enable thin LTO
- chromium: drop obsolete patch
    https://src.fedoraproject.org/rpms/chromium/c/7a9444bd5f608685cc1993c4cf8a3fee6825b084?branch=rawhide
- chromium: add llvm as Depends
- chromium: remove unused build argument
- chromium: build with libc++
- chromium: drop all bad-font-gc*.patch
    Effectively reverts https://dev.azure.com/AOSC-Tracking/chromium/_git/chromium/commit/3e4eb2d60bdffa5292b1374537f62895c9c909cb/ and https://dev.azure.com/AOSC-Tracking/chromium/_git/chromium/commit/f3c094d3ede83f2431448cd93be096c4d61c94ef/.
- chromium: update to 127.0.6533.88
    - Track patches at https://dev.azure.com/AOSC-Tracking/_git/chromium @ aosc/v127.0.6533.88.

... and 3 more commits

Package(s) Affected
-------------------

- chromium: 127.0.6533.99
- libyuv: 0+git20240215-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit libyuv chromium
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

Known Issues
------------------

- SIGILL on LoongArch64 <https://github.com/AOSC-Dev/aosc-os-abbs/pull/7270#issuecomment-2261873227>
- Sometimes NTP keeps loading forever, showing only a blank screen

References
------------------

- <https://issues.chromium.org/issues/356038456>
- <https://issues.chromium.org/issues/359039635>
- <https://src.fedoraproject.org/rpms/chromium>
- <https://salsa.debian.org/chromium-team/chromium/>
- <https://gitlab.archlinux.org/archlinux/packaging/packages/chromium>

### CODE REVIEW

**Comment by KexyBiscuit** [comment]
FTBFS on (but not limited to) AMD64.

https://issues.chromium.org/issues/346690214

**Comment by KexyBiscuit** [comment]
https://src.fedoraproject.org/rpms/chromium/c/5b27efc518c86b93dd2f30e877536599ed07beac

**Comment by KexyBiscuit** [comment]
LoongArch64 FTBFS.
```
[61964/61964] "/usr/bin/python3" "../../build/toolchain/gcc_link_wrapper.py" --output="./chrome" -- clang++ -Wl,--version-script=../../build/linux/chrome.map -fuse-ld=lld -Wl,--build-id=sha1 -fPIC -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now -Wl,--icf=all -Wl,--color-diagnostics -Wl,--undefined-version -Wl,--no-call-graph-profile-sort --target=loongarch64-linux-gnu -no-canonical-prefixes -Wl,-O2 -Wl,--gc-sections -Wl,--gdb-index -Wl,-z,defs -Wl,--as-needed -rdynamic -pie -Wl,--disable-new-dtags prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libstd.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/liballoc.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libcfg_if.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libcompiler_builtins.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libcore.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libgetopts.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libhashbrown.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/liblibc.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libpanic_abort.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libpanic_unwind.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/librustc_demangle.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libstd_detect.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libtest.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libunicode_width.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libunwind.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libaddr2line.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libadler.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libgimli.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libmemchr.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libminiz_oxide.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libobject.rlib -Wl,-O1,--sort-common,--as-needed -Wl,-build-id=sha1 -Wl,-z,relro -Wl,-z,now -fPIC -fPIE -pie  -fuse-linker-plugin -fPIC -o "./chrome" -Wl,--start-group @"./chrome.rsp" -Wl,--end-group   /usr/lib/clang/18/lib/linux/libclang_rt.builtins-loongarch64.a -latomic -ldl -lpthread -lrt -lgmodule-2.0 -lgobject-2.0 -lgthread-2.0 -lglib-2.0 -lsmime3 -lnss3 -lnssutil3 -lplds4 -lplc4 -lnspr4 -ldbus-1 -latk-1.0 -latk-bridge-2.0 -lcups -lgio-2.0 -ldrm -latspi -lresolv -lexpat -luuid -lm -lz -lX11 -lXcomposite -lXdamage -lXext -lXfixes -lXrender -lXrandr -lXtst -lxcb -lxkbcommon -lgbm -lXi -lpci -lffi -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -lcairo -lasound -lpulse -lstdc++ -lxshmfence -Wl,--start-group obj/third_party/rust/cxx/v1/lib/libcxx_lib.rlib obj/third_party/rust/serde_json_lenient/v0_2/wrapper/libthird_uparty_srust_sserde_ujson_ulenient_sv0_u2_swrapper_cwrapper.rlib obj/third_party/rust/serde/v1/lib/libserde_lib.rlib obj/third_party/rust/serde_json_lenient/v0_2/lib/libserde_json_lenient_lib.rlib obj/third_party/rust/itoa/v1/lib/libitoa_lib.rlib obj/third_party/rust/ryu/v1/lib/libryu_lib.rlib obj/build/rust/chromium_prelude/libchromium.rlib obj/skia/libskia_cbridge_urust_uside.rlib obj/third_party/rust/font_types/v0_5/lib/libfont_types_lib.rlib obj/third_party/rust/bytemuck/v1/lib/libbytemuck_lib.rlib obj/third_party/rust/read_fonts/v0_19/lib/libread_fonts_lib.rlib obj/third_party/rust/skrifa/v0_19/lib/libskrifa_lib.rlib obj/third_party/blink/common/rust_crash/libthird_uparty_sblink_scommon_srust_ucrash_crs.rlib obj/third_party/crabbyavif/libthird_uparty_scrabbyavif_ccrabbyavif.rlib obj/third_party/crabbyavif/libdav1d_sys.rlib obj/third_party/crabbyavif/liblibyuv_sys.rlib obj/third_party/rust/libc/v0_2/lib/liblibc_lib.rlib obj/components/qr_code_generator/libcomponents_sqr_ucode_ugenerator_cqr_ucode_ugenerator_uffi_uglue.rlib obj/third_party/rust/qr_code/v2/lib/libqr_code_lib.rlib -Wl,--end-group 
FAILED: chrome 
"/usr/bin/python3" "../../build/toolchain/gcc_link_wrapper.py" --output="./chrome" -- clang++ -Wl,--version-script=../../build/linux/chrome.map -fuse-ld=lld -Wl,--build-id=sha1 -fPIC -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now -Wl,--icf=all -Wl,--color-diagnostics -Wl,--undefined-version -Wl,--no-call-graph-profile-sort --target=loongarch64-linux-gnu -no-canonical-prefixes -Wl,-O2 -Wl,--gc-sections -Wl,--gdb-index -Wl,-z,defs -Wl,--as-needed -rdynamic -pie -Wl,--disable-new-dtags prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libstd.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/liballoc.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libcfg_if.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libcompiler_builtins.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libcore.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libgetopts.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libhashbrown.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/liblibc.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libpanic_abort.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libpanic_unwind.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/librustc_demangle.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libstd_detect.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libtest.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libunicode_width.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libunwind.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libaddr2line.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libadler.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libgimli.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libmemchr.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libminiz_oxide.rlib prebuilt_rustc_sysroot/lib/rustlib/loongarch64-unknown-linux-gnu/lib/libobject.rlib -Wl,-O1,--sort-common,--as-needed -Wl,-build-id=sha1 -Wl,-z,relro -Wl,-z,now -fPIC -fPIE -pie  -fuse-linker-plugin -fPIC -o "./chrome" -Wl,--start-group @"./chrome.rsp" -Wl,--end-group   /usr/lib/clang/18/lib/linux/libclang_rt.builtins-loongarch64.a -latomic -ldl -lpthread -lrt -lgmodule-2.0 -lgobject-2.0 -lgthread-2.0 -lglib-2.0 -lsmime3 -lnss3 -lnssutil3 -lplds4 -lplc4 -lnspr4 -ldbus-1 -latk-1.0 -latk-bridge-2.0 -lcups -lgio-2.0 -ldrm -latspi -lresolv -lexpat -luuid -lm -lz -lX11 -lXcomposite -lXdamage -lXext -lXfixes -lXrender -lXrandr -lXtst -lxcb -lxkbcommon -lgbm -lXi -lpci -lffi -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -lcairo -lasound -lpulse -lstdc++ -lxshmfence -Wl,--start-group obj/third_party/rust/cxx/v1/lib/libcxx_lib.rlib obj/third_party/rust/serde_json_lenient/v0_2/wrapper/libthird_uparty_srust_sserde_ujson_ulenient_sv0_u2_swrapper_cwrapper.rlib obj/third_party/rust/serde/v1/lib/libserde_lib.rlib obj/third_party/rust/serde_json_lenient/v0_2/lib/libserde_json_lenient_lib.rlib obj/third_party/rust/itoa/v1/lib/libitoa_lib.rlib obj/third_party/rust/ryu/v1/lib/libryu_lib.rlib obj/build/rust/chromium_prelude/libchromium.rlib obj/skia/libskia_cbridge_urust_uside.rlib obj/third_party/rust/font_types/v0_5/lib/libfont_types_lib.rlib obj/third_party/rust/bytemuck/v1/lib/libbytemuck_lib.rlib obj/third_party/rust/read_fonts/v0_19/lib/libread_fonts_lib.rlib obj/third_party/rust/skrifa/v0_19/lib/libskrifa_lib.rlib obj/third_party/blink/common/rust_crash/libthird_uparty_sblink_scommon_srust_ucrash_crs.rlib obj/third_party/crabbyavif/libthird_uparty_scrabbyavif_ccrabbyavif.rlib obj/third_party/crabbyavif/libdav1d_sys.rlib obj/third_party/crabbyavif/liblibyuv_sys.rlib obj/third_party/rust/libc/v0_2/lib/liblibc_lib.rlib obj/components/qr_code_generator/libcomponents_sqr_ucode_ugenerator_cqr_ucode_ugenerator_uffi_uglue.rlib obj/third_party/rust/qr_code/v2/lib/libqr_code_lib.rlib -Wl,--end-group 
ld.lld: error: /usr/bin/../lib64/gcc/loongarch64-unknown-linux-gnu/13.2.0/crtbeginS.o:(.text+0xfc): relocation R_LARCH_B26 out of range: 222057860 is not in [-134217728, 134217727]; references '__cxa_finalize'
>>> referenced by crtstuff.c
>>> defined in /usr/lib/libc.so.6
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by KexyBiscuit** [comment]
https://github.com/AOSC-Dev/aosc-os-abbs/pull/7270#issuecomment-2253625127 points to an issue related to https://gcc.gnu.org/onlinedocs/gcc/LoongArch-Options.html#index-mcmodel_003d-1, setting `-mcmodel=medium` or `-mcmodel=extreme` for the world _might_ fix the problem.

For internal packagers: https://t.me/c/1109254909/463057

File: core-devel/gcc/01-runtime/prepare [hunk 1]
```diff
@@ -22,6 +22,12 @@ source "$AB"/arch/${CROSS:-$ARCH}.sh
 abinfo 'GCC only passes CPPFLAGS in some stages, appending other flags to $CPPFLAGS ...'
 export CPPFLAGS="${CPPFLAGS} -O3 ${CFLAGS_COMMON_ARCH}"
 
+if ab_match_arch loongarch64; then
+    abinfo "Setting code model to medium …"
```
Comment (MingcongBai) [inline core-devel/gcc/01-runtime/prepare]: Let's just use three dots (...) for abinfo, ellipses look ugly in Monospace.

File: core-devel/gcc/spec [hunk 2]
```diff
@@ -1,5 +1,5 @@
 VER=13.2.0
-REL=6
+REL=7
```
Comment (MingcongBai) [inline core-devel/gcc/spec]: You'd need to bump aosc-aaa for this.

**Review by MingcongBai — DISMISSED** [review]

**Comment by KexyBiscuit** [comment]
Waiting for https://buildit.aosc.io/pipelines/8278 to complete.

**Comment by OriginCode** [comment]
Crashing when trying to login to Bitwarden extension and when it's starting doing webauthn authentication process:
```
[0731/200529.415173:ERROR:exception_snapshot_linux.cc(427)] invalid magic number 0x2f6a8100
[3663:3663:0731/200529.402595:ERROR:check.cc(376)] Check failed: false. NOTREACHED log messages are omitted in official builds. Sorry!
[3663:3663:0731/200530.173059:ERROR:object_proxy.cc(576)] Failed to call method: org.kde.KWallet.isEnabled: object_path= /modules/kwalletd5: org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying
[3663:3663:0731/200530.173109:ERROR:kwallet_dbus.cc(112)] Error contacting kwalletd5 (isEnabled)
[3663:3663:0731/200530.175222:ERROR:object_proxy.cc(576)] Failed to call method: org.kde.KLauncher.start_service_by_desktop_name: object_path= /KLauncher: org.freedesktop.DBus.Error.ServiceUnknown: The name org.kde.klauncher was not provided by any .service files
[3663:3663:0731/200530.175264:ERROR:kwallet_dbus.cc(81)] Error contacting klauncher to start kwalletd5
[3663:3663:0731/200530.633212:ERROR:object_proxy.cc(576)] Failed to call method: org.kde.KWallet.close: object_path= /modules/kwalletd5: org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying
[3663:3663:0731/200530.633261:ERROR:kwallet_dbus.cc(502)] Error contacting kwalletd5 (close)
[3663:3663:0731/200544.446428:FATAL:check.cc(376)] Check failed: false. NOTREACHED log messages are omitted in official builds. Sorry!
[0731/200544.475928:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0731/200544.476210:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0731/200544.476478:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0731/200544.476614:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0731/200544.476863:ERROR:elf_dynamic_array_reader.h(64)] tag not found
[0731/200544.494755:ERROR:exception_snapshot_linux.cc(427)] invalid magic number 0x41535801
[3702:3708:0731/200546.390078:ERROR:ssl_client_socket_impl.cc(883)] handshake failed; returned -1, SSL error code 1, net_error -3
fish: Job 1, 'chromium' terminated by signal SIGILL (Illegal instruction)
```

**Comment by OriginCode** [comment]
> Crashing when trying to login to Bitwarden extension and when it's starting doing webauthn authentication process:
> 
> ```
> [0731/200529.415173:ERROR:exception_snapshot_linux.cc(427)] invalid magic number 0x2f6a8100
> [3663:3663:0731/200529.402595:ERROR:check.cc(376)] Check failed: false. NOTREACHED log messages are omitted in official builds. Sorry!
> [3663:3663:0731/200530.173059:ERROR:object_proxy.cc(576)] Failed to call method: org.kde.KWallet.isEnabled: object_path= /modules/kwalletd5: org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying
> [3663:3663:0731/200530.173109:ERROR:kwallet_dbus.cc(112)] Error contacting kwalletd5 (isEnabled)
> [3663:3663:0731/200530.175222:ERROR:object_proxy.cc(576)] Failed to call method: org.kde.KLauncher.start_service_by_desktop_name: object_path= /KLauncher: org.freedesktop.DBus.Error.ServiceUnknown: The name org.kde.klauncher was not provided by any .service files
> [3663:3663:0731/200530.175264:ERROR:kwallet_dbus.cc(81)] Error contacting klauncher to start kwalletd5
> [3663:3663:0731/200530.633212:ERROR:object_proxy.cc(576)] Failed to call method: org.kde.KWallet.close: object_path= /modules/kwalletd5: org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying
> [3663:3663:0731/200530.633261:ERROR:kwallet_dbus.cc(502)] Error contacting kwalletd5 (close)
> [3663:3663:0731/200544.446428:FATAL:check.cc(376)] Check failed: false. NOTREACHED log messages are omitted in official builds. Sorry!
> [0731/200544.475928:ERROR:elf_dynamic_array_reader.h(64)] tag not found
> [0731/200544.476210:ERROR:elf_dynamic_array_reader.h(64)] tag not found
> [0731/200544.476478:ERROR:elf_dynamic_array_reader.h(64)] tag not found
> [0731/200544.476614:ERROR:elf_dynamic_array_reader.h(64)] tag not found
> [0731/200544.476863:ERROR:elf_dynamic_array_reader.h(64)] tag not found
> [0731/200544.494755:ERROR:exception_snapshot_linux.cc(427)] invalid magic number 0x41535801
> [3702:3708:0731/200546.390078:ERROR:ssl_client_socket_impl.cc(883)] handshake failed; returned -1, SSL error code 1, net_error -3
> fish: Job 1, 'chromium' terminated by signal SIGILL (Illegal instruction)
> ```

Reproduced on 127.0.6533.72.
Works fine on 126.0.6478.182.

Possible 127. regression.

**Comment by KexyBiscuit** [comment]
Chromium shows a blank screen during OOBE on AMD64 and AArch64. The screenshot from 126 is what expected to be shown.
image

**Comment by KexyBiscuit** [comment]
FTBFS on LoongArch64, cxxbridge exits with `Segmentation fault (core dumped)`.
```
[2654/61969] /usr/bin/python3 ../../third_party/rust/cxx/chromium_integration/run_cxxbridge.py --exe default_for_rust_host_build_tools/cxxbridge --header gen/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs.h --cc gen/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs.cc ../../third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs --
FAILED: gen/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs.h gen/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs.cc 
/usr/bin/python3 ../../third_party/rust/cxx/chromium_integration/run_cxxbridge.py --exe default_for_rust_host_build_tools/cxxbridge --header gen/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs.h --cc gen/third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs.cc ../../third_party/rust/serde_json_lenient/v0_2/wrapper/lib.rs --
```

**Comment by KexyBiscuit** [comment]
Google has already released [Early Stable Release](https://chromium.googlesource.com/chromium/src.git/+/main/docs/process/release_cycle.md#stable-promotion) of Chromium 128, make this pull request ready for review now, requesting waiver for the known issues.

**Comment by KexyBiscuit** [comment]
Also, Chromium 127.0.6533.119 has just been released, which will be worked on after this pull request is merged, since that version is not a security update (but this pull request is).

**Comment by MingcongBai** [comment]
> Google has already released [Early Stable Release](https://chromium.googlesource.com/chromium/src.git/+/main/docs/process/release_cycle.md#stable-promotion) of Chromium 128, make this pull request ready for review now, requesting waiver for the known issues.

Due diligence served I think and it works well enough for AMD64. Approving.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**